docs/submission/submitting-clinical-data.md

```diff
@@ -57,8 +57,8 @@ Uploaded data must be checked for accuracy against the data in the current submi
 
 ![Sign off Submission](/assets/submission/clinical-4-signoff.png)
 
-1. Once you have reviewed the files in the file preview area, click the **Sign Off Submission** button to submit your clinical data.
+1. Once you have reviewed the files in the file preview area and your submission is valid, click the **Sign Off Submission** button to submit your clinical data.
 1. If you have made any updates to already submitted data, your clinical workspace will be locked in a Pending Approval state. A DCC member will review your updates and approve them for submission. If there are any issues, your submission will be reopened and the DCC will notify you, via email, of the required changes.
 1. If your data does not require approval by DCC, it will be submitted automatically.
 
-Once submitted, you will see the clincial data reflected on your [Program Dashboard](/docs/submitted-data).
+Once submitted, you will see the clincial data reflected on your [Program Dashboard](submitted-data).
```

Comment:
i think this needs to folder link? `/submission/submitted-data`

Reply:
yes I will change this